Library/Formula/libspatialite.rb

```diff
@@ -38,6 +38,13 @@ class Libspatialite < Formula
   depends_on "freexl" => :recommended
   depends_on "liblwgeom" => :recommended
 
+  if build.stable?
+    patch do
```

Comment:
Belongs in a `stable do` block with the stable url/checksum rather than `if build.stable?` :wink: